doc/Administration-guide/System-components-overview/index.md

```diff
@@ -5,63 +5,73 @@ permalink: /:path/
 sort_idx: 10
 ---
 
-Guide for the platform administrators.
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
 
+* TOC
+{:toc}
 
-* [High-level architecture](#high-level-architecture)
-  * [Kaa cluster node](#kaa-cluster-node)
-    * [Control service](#control-service)
-    * [Operations service](#operations-service)
-    * [Bootstrap service](#bootstrap-service)
-  * [Endpoint](#endpoint)
-  * [Third-party](#third-party)
-    * [Zookeeper](#zookeeper)
-    * [SQL database](#sql-database)
-    * [NoSQL database](#nosql-database)
-* [Further reading](#further-reading)
+The Kaa IoT platform consists of the Kaa server, Kaa extensions, and endpoint SDKs (see [Architecture overview]({{root_url}}Architecture-overview/) for more details).
```

Comment:
According to our "Documentation contribution and style guide" something like "This guide will provide the overview of key system components from the administration point of view" is required.